contract_show_invoice/__manifest__.py

```diff
@@ -1,20 +1,22 @@
 # -*- coding: utf-8 -*-
 # © 2015 Angel Moya <angel.moya@domatix.com>
 # © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# © 2017 Dave Burkholder <dave@thinkwelldesigns.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Contract Show Invoice',
     'summary': 'Button in contracts to show their invoices',
-    'version': '9.0.1.1.0',
+    'version': '10.0.1.0.0',
     'author': 'Domatix,'
               'Tecnativa,'
+              'Thinkwell Designs,'
```

Comment:
Oops I commented on an outdated diff, so this was collapsed. Commenting on current:

Authorship and copyright is a bit of an overreach for the work that was performed here. Please limit the attribution to the ReadMe in this instance